generator/generator.yml

```diff
@@ -218,3 +218,14 @@ modules:
       - interfaces
       - ifXTable
       - 1.3.6.1.4.1.41112.1.6 # ubntUniFi
+
+# MiktoTik
+#
+# http://download2.mikrotik.com/Mikrotik.mib
+#
+  mikrotik:
+    walk:
+      - sysUpTime
+      - interfaces
+      - ifXTable
+      - 1.3.6.1.4.1.14988.1.1
```

Comment:
What's this object's name?

Reply:
`snmptranslate` gives me this:
```
.iso.org.dod.internet.private.enterprises.mikrotik.mikrotikExperimentalModule.mtXRouterOs
```

Reply:
We don't usually pull in entire MIBs. Can you be more specific about which parts of it you want?